content/How an engineer can be your best partner.md

```diff
@@ -0,0 +1,86 @@
+# **GROWTH - Data Analysis based decisions**
+
+![main image](images/growth.jpg)
+
+Have you ever said one of these tricky phrases?
+  - I'm sure this feature is going to rock it!
+  - We need this feature before going live, it is a must!
+  - Users will use this for sure!
+
+Well, if you found yourself saying any of those without any numbers that support you, you may probably be making a lot of assumptions.
+So is like you are playing roulette, due to lacking data to prove your hypothesis.
+Give growth a try! And find out how to move smarter with your development team along with really learning what your users want!
+
+## 1) What's GROWTH
```

Comment:
Are you missing a `?` here? Also, is the numbering necessary?

Reply:
Agreed about the numbers, they look weird

Reply:
I was also waiting for a GROWTH explanation but the content it's all about data analysis

Reply:
I see this as a statement, not a question, would it be more accurate to change it?
Regarding numbering... just a personal touch 😄